ui/partials/main/thShortcutTable.html

```diff
@@ -1,4 +1,4 @@
-<div class="card">
+<div class="card" style="max-height: 90vh; overflow: auto">
   <div class="card-header">
```

Comment:
Saw this in passing, I defer to whoever reviews this :) But I think the typical location for properties would be in our various css files. For example in this case we target the onscreen shortcut overlay here, in treeherder-global.css.
https://github.com/mozilla/treeherder/blob/51dc3cc938234c441c6b95a9715ac0daf5f03f9c/ui/css/treeherder-global.css#L393-L403

Reply:
Thanks for the comment! I wasn't sure where the CSS should go which is why I just put it inline. I've updated the patch to add an id to the element, and moved the css to treeherder-global.css.

Reply:
Not sure, but I think you may be able to add your properties directly to `#onscreen-overlay` rather than create a new class. Did you try that?

If it absolutely has to be on that child div, maybe we could keep the same nomenclature, and have `#onscreen-overlay-card`.

Reply:
> If it absolutely has to be on that child div, maybe we could keep the same nomenclature, and have #onscreen-overlay-card.

Reason, since we grab the same shortcut table partial for our Help > User Guide, and my proposed name `#onscreen-overlay-card` would differentiate / tie the class to the onscreen presentation only.